pyproject.toml

```diff
@@ -52,19 +52,18 @@ test = ["pytest>=7.2"]
 coverage = ["mqt.predictor[test]", "pytest-cov>=4"]
 dev = ["mqt.predictor[coverage]"]
 docs = [
-    "furo>=2023.08.17",
-    "sphinx>=7,<9",
-    "setuptools-scm>=7",
+    "furo>=2023.9.10",
+    "setuptools-scm>=8.1",
+    "sphinx_design>=0.6",
+    "sphinx-autoapi>=3",
+    "sphinx-copybutton>=0.5",
     "sphinxcontrib-bibtex>=2.4.2",
-    "sphinx-copybutton",
-    "sphinx-hoverxref",
```

Comment:
Based on this removal, you will most likely also need to adapt the docs/conf.py and remove the hoverxref config. Missed this in qcec and it made the docs build fail.
Worth running `nox -s docs` locally to test

Reply:
Adjusted it.